desktop-widgets/tab-widgets/TabDiveInformation.cpp

```diff
@@ -93,9 +92,11 @@ void TabDiveInformation::closeWarning()
 }
 
 void TabDiveInformation::updateWaterTypeWidget()
-{    // Decide on whether to show the water type/salinity combobox or not
-	if (prefs.salinityEditDefault || manualDive)
-	{         // if the preference setting has been checked or this is a manually-entered dive
+{
+    bool hasDCSalinity = current_dive && current_dive->salinity != 0;
+    // Decide on whether to show the water type/salinity combobox or not
+	if (prefs.salinityEditDefault || !hasDCSalinity)
```

Comment:
Coding style: brace on same line as `if`.
I reckon that the code was like that before, but since you're changing it, might as well fix this.

Reply:
Done! https://github.com/rafsalvioni/subsurface/commit/9334c8b1d45eaebd7706c1d7b605cd414906e8b7